### ISSUE TITLE
Buttons: Text inside the button doesn't match the title (icon only)

### ISSUE THREAD

**Issue by ZeroBrushV2** [primefaces/primefaces#9738]
### Describe the bug

Using the component MenuButton in its variation IconOnly (not using `value` attribute) PrimeFaces generates the following html:
```
<span id="j_idt343:j_idt345" class="ui-menubutton ui-widget">
<button id="j_idt343:j_idt345_button" name="j_idt343:j_idt345_button" type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-text-icon-left" role="button" aria-disabled="false">
  <span class="ui-button-icon-left ui-icon ui-c ui-icon-triangle-1-s"></span>
  <span class="ui-button-text ui-c">ui-button</span>
</button>
</span>
```

This causes a problem with the screen reader that will ignore the `title` or `aria-label` and will spell "ui-button".

The are two problems with the generation:

- `Title` attribute will be ignored and not generated
- For the screen reader `aria-label` and `title` are overridden by `<span class="ui-button-text ui-c">ui-button</span></button></span>`

### Reproducer

Use menuButton without `value` attribute, to generate a button with icon only

### Expected behavior

PrimeFaces should consider the following during the generation of the element:
- `Title` and `aria-label` should override the text inside the `<span class="ui-button-text ui-c">`
- `title` should be considered as the primary text (ONLY IN CASE OF ICON ONLY MENUBUTTON, open to consideration),
**NOTE: If the `value` is not compiled the text generated should still be invisible to a normal user**

Example:
Using the following code:
```
	<p:menuButton  title="My Button That Does Stuff" icon="fa fa-bars">
			<p:menuitem >
			</p:menuitem>
			<p:menuitem>
			</p:menuitem>
			<p:menuitem >
			</p:menuitem>
	</p:menuButton>
```

Should generate the following html:
```
<span id="j_idt343:j_idt345" class="ui-menubutton ui-widget">
<button id="j_idt343:j_idt345_button" name="j_idt343:j_idt345_button" type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-text-icon-left" role="button" aria-disabled="false">
  <span class="ui-button-icon-left ui-icon ui-c ui-icon-triangle-1-s"></span>
  <span class="ui-button-text ui-c">My Button That Does Stuff</span>
</button>
</span>
```

### PrimeFaces edition

Community

### PrimeFaces version

12.0.0

### JSF implementation

MyFaces

### JSF version

2.3

### Java version

1.8

**Comment by melloware** [comment]
Thanks the same issue was in all Icon Only buttons like...

* Button
* CommandButton
* LinkButton
* MenuButton
* SplitButton
* SelectBooleanButton
